src/routes/resources/parts-of-a-shinai-and-how-to-hold-it/+page.svelte

```diff
@@ -0,0 +1,256 @@
+		fingers are shown to be relaxed, but not open.
+	</p>
+</div>
+
+<div class="grid-container">
+	<figure>
+		<enhanced:img
+			src="./both-hands-shinai-grip-top-view.jpg?w=400;350;200"
+			sizes="(min-width:1280px) 400px,(min-width:450px) 350px, (min-width:0px) 200px"
+			alt="View from above of gripping a shinai, as if looking down at one's own hands. Dotted lines show a V shape made between the thumb and index finger of each hands, and the V shapes lining up along the shinai handle."
+		/>
+	</figure>
+
+	<div>
+		<p>
+			At any point during cutting, you should be able to open your fingers and they should be
+			pointing directly away from you at every moment in cutting.
+		</p>
+
+		<p>
+			In bogu it is difficult to see correct placement of the hands, because of the kote. But your
+			cuts will reflect to everyone else if you positioning is incorrect. It is important to always
+			be aware of correct grip on the shinai. Check yourself, until it becomes second nature to you.
+		</p>
+	</div>
+</div>
+
+<div class="grid-container">
+	<figure>
+		<enhanced:img
+			src="./both-hands-shinai-grip-mistakes.jpg?w=400;350;200"
+			sizes="(min-width:1280px) 400px,(min-width:450px) 350px, (min-width:0px) 200px"
+			alt="Three examples of incorrect shinai grips, two from above and one from the side. First and second examples show baseball bat grips. Third example shows left hand a few inches up from the end of shinai handle."
+		/>
+	</figure>
+
+	<p>
+		Make every effort to avoid gripping the shinai as shown in the pictures of common mistakes. They
+		are all excellent examples of what not to do. When gripping the shinai correctly, you will
+		naturally develop a strong kamae and be much less open to attacks.
+	</p>
+</div>
+
+<h3>Point Your Shinai</h3>
+<div class="grid-container">
+	<figure>
+		<enhanced:img
+			src="./both-hands-shinai-grip-in-bogu-front-view.jpg?w=400;350;200"
+			sizes="(min-width:1280px) 400px,(min-width:450px) 350px, (min-width:0px) 200px"
+			alt="Kendo practitioner in full armor, holding a shinai and facing forward. Demonstrates the angle of the shinai and where the tip points."
+		/>
+	</figure>
+
+	<p>
+		Now that your hands are gripping the shinai properly, let's focus on the final aspect of holding
+		the shinai. The tip of the shinai, the kissaki, is always pointed toward your opponent's throat.
+		If the two of you walked directly towards each other, your kissaki would end up touching each
+		other's throat. When practicing alone, picture a perfect reflection of yourself as your target
+		and adjust the height of the tip of the shinai accordingly. It can help to practice facing a
+		mirror.
+	</p>
+</div>
+
+<p>
+	If you have any further questions, feel free to address them to your senpai or email them to us.
+</p>
+
+<style>
+	figure {
+		margin-bottom: 15px;
+		display: table;
+		/* max-width: 47%; */
+	}
+	figcaption {
+		display: table-caption;
```

Comment:
why do we need all this table stuff?

Reply:
Without it, the captions mess with image centering and look bad on certain smaller screen sizes. Let's talk about this one in-person and see if we can find a better way.
![image](https://github.com/user-attachments/assets/5909e756-6a18-4516-b2e3-7a16d8b04966)

Reply:
Fixed when I fixed the awkward image sizing issue [78dc44d](https://github.com/kendoclub-at-umich/website/pull/199/commits/78dc44dac43175039c9d03c463b3dc597158d16d)

Reply:
Yay!